CONTRIBUTING.md

```diff
@@ -245,6 +245,74 @@ If you are adding a new service directory, ensure that it is mapped to a friendl
 8. Copy existing generate.ps1 file from another service and update the `ResourceProvider` name that is applicable to your SDK. Resource provider refers to the relative path of your REST spec directory in Azure-Rest-Api-Specs repository
    During SDK generation, this path helps to locate the REST API spec from the `https://github.com/Azure/azure-rest-api-specs`
 
+# On-boarding New generated code library
+
+1. Make a copy of `/sdk/template/Azure.Template` in you appropriate service directory and rename projects to `Azure.Management.*`. (e.g.  `sdk/storage/Azure.Management.Storage`)
```

Comment:
Why Azure.Management? We might be generating non-management libraries too.

Reply:
Wrote this for folks trying management SDKs initially.